lib/ex_okex/futures/private.ex

```diff
@@ -14,11 +14,20 @@ defmodule ExOkex.Futures.Private do
 
   ## Examples
 
-      iex> ExOkex.Futures.create_order(%{"instrument_id":"BTC-USD-180213","type":"1","price":"432.11","size":"2","match_price":"0","leverage":"10"})
-      # TODO: Add response sample
+  iex> ExOkex.Futures.create_order(%{
```

Comment:
Okex has a different params structure when using single orders vs bulk create. see https://www.okex.com/docs/en/#futures-orders

`orders_data` param is for bulk create